packages/tests/src.ts/test-providers.spec.ts

```diff
@@ -781,30 +759,30 @@ testFunctions.push({
         return false;
     },
     execute: async (provider: ethers.providers.Provider) => {
```

Comment:
We can comment it for now, but at some point we will need to execute real world tests against hedera testnet and the test will be very similar